Dockerfile

```diff
@@ -23,6 +23,13 @@ RUN apt-get install -y wget
 RUN mkdir -p /root/.dockstore/language-plugins
 RUN wget -P /root/.dockstore/language-plugins https://artifacts.oicr.on.ca/artifactory/collab-release/com/github/galaxyproject/dockstore-galaxy-interface/dockstore-galaxy-interface/${galaxy_plugin_version}/dockstore-galaxy-interface-${galaxy_plugin_version}.jar
 
+# Include snakemake language plugin
+ARG snakemake_plugin_version=0.0.3
+RUN apt-get install -y wget
```

Comment:
Isn't wget already installed on line 22?

Similarly, the `language-plugins` directory is also created on line 23

Reply:
We could move the `apt-get` and `mkdir` into separate "prepare to get plugins" block, above the galaxy download, to make it more clear and so that the galaxy download block and snakemake download block both look the same.